docs/aerodromes/classd/Archerfield.md

```diff
@@ -69,6 +69,64 @@ The circuit direction changes depending on time of day and runway being used.
 
 Circuits are to be flown at `A010`.
 
+## Helicopter Operations
+Archerfield has three helipads located around the aerodrome:
+
+- **Central Helipad**, on the southern side of Taxiway Foxtrot
+- **Northern Helipad**, adjacent the north-eastern end of Runway 04R/22L
+- The **Queensland Government Air facility** used by rescue helicopters, north of Taxiway B5
```

Comment:
The helipads appear to be called Central and Northern, but your notes refer to Northern and Southern pads. Should that be Central instead?

Reply:
Apologies I seemed southern pad was meant to say central will fix.